recipes/libffi/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+import os
+
+from conans import ConanFile, CMake
+
+
+class TclTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
```

Comment:
Are imports needed? running with `run_environment=True` should be enough, right?

Reply:
I don't think it's needed. just a leftover from old days before we invented `run_environment`